examples/test_cli.py

```diff
@@ -9,7 +9,7 @@
 def test_config_sample_2pop():
   os.chdir(os.path.dirname(__file__) + "/tutorials")
   runner = CliRunner()
-  result = runner.invoke(simulate, ['-c', 'config_sample_2pop.yml'])
+  result = runner.invoke(simulate, ['config_sample_2pop.yml'])
```

Comment:
they are - note line 33 below. I changed these to demonstrate the new capabilities, and left the last one to demo backwards compatibility.

Reply:
i've also added a test demonstrating the re-route of gempyor-simulate

Reply:
Fair enough. Let's chat later today about the right testing regime here - I don't think we should need to reach all the way into execution to confirm that the resulting configs / arguments passed are identical. Might warrant a bit of re-architecture / reframing of tests generally to do that, however.

Reply:
> I don't think we should need to reach all the way into execution to confirm that the resulting configs / arguments passed are identical.

Maybe, I think I would be more confident in that not being needed if there were other unit tests to accompany this PR.

Reply:
Agree - but I also think it's going to entail re-orienting the click approach a bit. Right now, parse happens in the commands, rather than in the main group, then dispatched down.

I think its definitely preferrable to centralize this even more (i.e. not just in one function, but into one call of that function), but that's maybe a more radical overhaul of the function signatures.